packages/webview_flutter/webview_flutter_wkwebview/ios/Classes/FWFHTTPCookieStoreHostApi.m

```diff
@@ -7,7 +7,9 @@
 #import "FWFWebsiteDataStoreHostApi.h"
 
 @interface FWFHTTPCookieStoreHostApiImpl ()
-@property(nonatomic) FWFInstanceManager *instanceManager;
+// BinaryMessenger and InstanceManager must be weak to prevent a circular reference
```

Comment:
You're adding a reference to BinaryMessenger in this comment throughout the files, even though many of the classes don't have BinaryMessenger properties.

Also, the comment is confusing now because ther are two subjects, but layer the comment refers to "the objects **it** stores", which applies only to one of them.

Reply:
Clarified the the comments and created separate ones for `BinaryMessenger` and `InstanceManager`.